### PR TITLE
21 use case run game

### PR DESCRIPTION
**Pull request by bac423**
Implemented without access to PGE use-case, but it should be fine

### CODE REVIEW

**Review by AlekseyPanas — CHANGES_REQUESTED** [review]
Interactor is perfect, just fix the tests

**Review by AlekseyPanas — APPROVED** [review]

**Review by cotopaxi64 — DISMISSED** [review]
Looks good, and incredibly well documented!

File: src/test/java/usecases/run_game/RgTaskTests.java [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+            this.gameOverValue = gameOverValue;
+        }
+
+        @Override
+        public Collection<Player> getPlayers() {
+            return players;
+        }
+
+        @Override
+        public void onTimerUpdate() {
```
Comment (joel-mat-03) [inline src/test/java/usecases/run_game/RgTaskTests.java:48]: write java docs for overriden methods except getters and setters

**Review by joel-mat-03 — CHANGES_REQUESTED** [review]
Documentation is good and all coding conventions are followed except for the one comment I have left regarding the java docs for the inner class CustomizableTestGame in RgTaskTests. Please make these changes.

**Review by joel-mat-03 — APPROVED** [review]
Looks good. Requested edits have been made and all conventions are followed.

**Review by cotopaxi64 — APPROVED** [review]
Looks good, and incredibly well documented!

**Pull request merged**